site/content/component/auth-server.md

```diff
@@ -127,5 +127,5 @@ The corresponding command to start up the server with the configuration used in
 
 {{% note %}}
 You may want to make logging of the server a little more verbose by enabling the *dev* Spring profile.
-To do so, append *,--spring.profiles.active=authentication-impl,dev* to the command line.
+To do so, append *-Dspring.profiles.active=authentication-impl,dev* to the command line.
```

Comment:
I agree that activating multiple profiles doesn't work the original way (because of the comma as separator). But I also think that what you propose doesn't work as well. The key needs to be `-Drun.profiles=...` if I am not mistaken (see Spring Boot Maven plugin documentation)